packages/rainbowkit/src/wallets/walletConnectors/walletConnectWallet/walletConnectWallet.ts

```diff
@@ -1,47 +1,31 @@
-import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
-import { getWalletConnectUri } from '../../../utils/getWalletConnectUri';
-import { isIOS } from '../../../utils/isMobile';
-import { Wallet } from '../../Wallet';
+import { RainbowKitWalletConnectParameters, Wallet } from '../../Wallet';
 import { getWalletConnectConnector } from '../../getWalletConnectConnector';
-import type { WalletConnectConnectorOptions } from '../../getWalletConnectConnector';
 
 export interface WalletConnectWalletOptions {
   projectId: string;
-  chains: Chain[];
-  options?: WalletConnectConnectorOptions;
+  options?: RainbowKitWalletConnectParameters;
 }
 
+// Used for maintance purposes for `connectorsForWallets` logic
+// wcId = "WalletConnect id"
+export const wcId = 'walletConnect';
+
 export const walletConnectWallet = ({
-  chains,
-  options,
   projectId,
-}: WalletConnectWalletOptions): Wallet => ({
-  id: 'walletConnect',
-  name: 'WalletConnect',
-  iconUrl: async () => (await import('./walletConnectWallet.svg')).default,
-  iconBackground: '#3b99fc',
-  createConnector: () => {
-    const ios = isIOS();
+  options,
+}: WalletConnectWalletOptions): Wallet => {
+  const getUri = (uri: string) => uri;
 
-    const connector = getWalletConnectConnector({
-      chains,
+  return {
+    id: wcId,
+    name: 'WalletConnect',
+    installed: true,
```

Comment:
`installed` should always be `undefined` for this connector (as it isn't eip-1193), right?